app/src/test/java/com/github/sdpteam15/polyevents/MockMatcher.kt

```diff
@@ -0,0 +1,89 @@
+package com.github.sdpteam15.polyevents
```

Comment:
What is the purpose of this file ? I'm not sure I get it

Reply:
get from master PR #142